server/utils/errorHandler/errorWrapper.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response, NextFunction, RequestHandler} from "express";
+
+const errorMappings: { [name: string]: number } ={
+  JsonWebTokenError: 401,
+  Unauthorized: 401,
+  ValidationError: 422,
+  "NotFound": 404
+};
+
+const errorWrapper = async(controller: RequestHandler,req:Request, res:Response, next:NextFunction) =>{
```

Comment:
Error Wrapper, needs to take just one Parameter, errorWrapper(controller: RequestHandler),
To use It in Routes Files like this : 
`
router.get('/my-route', errorWrapper(myController));
`

Reply:
done, will be solved in seconds